vector/src/main/res/xml/vector_settings_preferences.xml

```diff
@@ -82,7 +82,8 @@
             android:defaultValue="true"
```

Comment:
I forgot to ask you to turn this pref off, please set false by default
Please consider the default in VectorPreferences.kt too